source/src/protocols/ligand_docking/HighResDocker.cc

```diff
@@ -282,11 +291,26 @@ HighResDocker::apply(core::pose::Pose & pose) {
 	score_fxn_->score( pose ); // without this neither of the movers below were working
 	// I believe that this may have been related to adding constraints incorrectly at other places in my code.
 	// Rigid body exploration
-	utility::vector1<protocols::moves::MoverOP> rigid_body_movers= create_rigid_body_movers(pose);
+	utility::vector1<protocols::moves::MoverOP> rigid_body_movers = create_rigid_body_movers(pose);
+
+    core::Size ligand_index = 0;
+    for ( core::Size i(1), imax(pose.total_residue()); i<=imax; ++i ) {
+        core::conformation::Residue const & residue( pose.residue(i) );
+        if ( residue.is_ligand() ) {
+            ligand_index = i;
```

Comment:
There can theoretically be multiple ligands in the pose, only one of which (not necessarily the last) is the one which is being modified. Unfortunately, things are indirect enough that I don't think there's a good way of getting just the primary ligand from the info provided, so there likely isn't a need to change things. Just make sure to document the issue thoroughly. -- Though I might suggest breaking this out into a private function, which might make updating things later easier.